src/app/views/App.tsx

```diff
@@ -339,10 +347,27 @@ class App extends Component<IAppProps, IAppState> {
     if (mobileScreen) {
       sidebarWidth = layout = 'col-xs-12 col-sm-12';
     }
-
+    if (permissionModeType && this.state.hideDialog) {
+      this.showDialog();
+    }
+    const teamsapp = <a href={"https://www.bing.com/?form=000010"}>{translateMessage('Sample Explorer Teams app')}</a>;
+    //TODO: put in the url when we have this set up
```

Comment:
can you make sure to link an ADO task here please?

Reply:
4fc227788663221f1f5218f3faf39a92095fef83